target/linux/ar71xx/image/generic.mk

```diff
@@ -196,6 +209,45 @@ define Device/mr16
 endef
 TARGET_DEVICES += mr16
 
+define Device/mr1750
+  DEVICE_TITLE := OpenMesh MR1750
+  DEVICE_PACKAGES := om-watchdog kmod-ath10k ath10k-firmware-qca988x
+  BOARDNAME = MR1750
+  FILESYSTEMS := squashfs
+  BLOCKSIZE := 4 8 16 64 128 256
+  KERNEL := kernel-bin | lzma | uImage lzma
+  IMAGES := sysupgrade.bin factory.bin
+  IMAGE/sysupgrade.bin/squashfs := append-rootfs | pad-rootfs | combined-ext-image board=MR1750 rootfs=$$$$@
```

Comment:
Looks to me like it is fs-types-y by default. And this can be squashfs, ext4, iso, ...

Enabling ext4 without overwriting "FILESYSTEMS := squashfs" will also create a non-working ext4 image for this board or let the build fail. 

My current test setup will simple fail to build when removing the FILESYSTEMS line:

    CONFIG_TARGET_ar71xx=y
    CONFIG_TARGET_ar71xx_generic=y
    CONFIG_TARGET_MULTI_PROFILE=y
    CONFIG_TARGET_DEVICE_ar71xx_generic_DEVICE_a60=y
    CONFIG_TARGET_DEVICE_ar71xx_generic_DEVICE_mr1750=y
    CONFIG_TARGET_DEVICE_ar71xx_generic_DEVICE_mr600=y
    CONFIG_TARGET_DEVICE_ar71xx_generic_DEVICE_mr900=y
    CONFIG_TARGET_DEVICE_ar71xx_generic_DEVICE_om2p=y
    CONFIG_TARGET_DEVICE_ar71xx_generic_DEVICE_om5pac=y
    CONFIG_TARGET_DEVICE_ar71xx_generic_DEVICE_om5p=y
    CONFIG_KMOD_BATMAN_ADV_BATMAN_V=y
    CONFIG_KMOD_BATMAN_ADV_BLA=y
    CONFIG_KMOD_BATMAN_ADV_DAT=y
    CONFIG_KMOD_BATMAN_ADV_DEBUGFS=y
    CONFIG_KMOD_BATMAN_ADV_MCAST=y
    CONFIG_PACKAGE_kmod-batman-adv=y
    CONFIG_PACKAGE_kmod-crypto-crc32c=y
    CONFIG_PACKAGE_kmod-crypto-hash=y
    CONFIG_PACKAGE_kmod-lib-crc16=y
    CONFIG_PACKAGE_kmod-lib-crc32c=y
    CONFIG_TARGET_EXT4_BLOCKSIZE=4096
    CONFIG_TARGET_EXT4_BLOCKSIZE_4K=y
    CONFIG_TARGET_EXT4_RESERVED_PCT=0
    CONFIG_TARGET_IMAGES_GZIP=y
    CONFIG_TARGET_ROOTFS_EXT4FS=y
    CONFIG_TARGET_ROOTFS_PARTSIZE=256

Here the interesting part of the build:

    /home/sven/tmp/lede/staging_dir/host/bin/padjffs2 /home/sven/tmp/lede/build_dir/target-mips_24kc_musl/linux-ar71xx_generic/tmp/lede-ar71xx-generic-a60-ext4-sysupgrade.bin  64
    padding image to 10150000
    WARNING: Image file /home/sven/tmp/lede/build_dir/target-mips_24kc_musl/linux-ar71xx_generic/tmp/lede-ar71xx-generic-a60-ext4-sysupgrade.bin is too big
    gzip -c -9n /home/sven/tmp/lede/build_dir/target-mips_24kc_musl/linux-ar71xx_generic/tmp/lede-ar71xx-generic-a60-ext4-sysupgrade.bin > /home/sven/tmp/lede/bin/targets/ar71xx/generic/lede-ar71xx-generic-a60-ext4-sysupgrade.bin.gz
    gzip: /home/sven/tmp/lede/build_dir/target-mips_24kc_musl/linux-ar71xx_generic/tmp/lede-ar71xx-generic-a60-ext4-sysupgrade.bin: No such file or directory
    Makefile:113: recipe for target '/home/sven/tmp/lede/bin/targets/ar71xx/generic/lede-ar71xx-generic-a60-ext4-sysupgrade.bin.gz' failed


And when also removing the /squashfs suffix, the resulting image is not usable on the devices.

So in theory, the /squashfs suffix could be removed when the "FILESYSTEM" is kept. But this would also mean that there is a line which can only create valid squashfs image which is not marked that way.